.travis.yml

```diff
@@ -58,7 +58,7 @@ before_install:
   - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
 
 install:
-  - travis_retry composer install $COMPOSER_ARGS --ignore-platform-reqs
+  - travis_retry composer install $COMPOSER_ARGS
```

Comment:
Revert this and add `paragonie/random_compat` into `LEGACY_DEPS` on PHP 5.6 locked

Reply:
@webimpress :+1: